pkgs/tools/networking/gandi-cli/default.nix

```
@@ -5,10 +5,10 @@ with pythonPackages;
 buildPythonPackage rec {
   namePrefix = "";
   name = "gandi-cli-${version}";
```

```suggestion
 pname = "gandi-cli";
```
`buildPython*` will put `pname` and `version` together to make `name`.